### PR TITLE
Add the new functionality in MoreViewController and update pod …

### PR DESCRIPTION
**Pull request by baquer**
resolve all the conflicts of PR #57

### CODE REVIEW

**Comment by baquer** [comment]
@chashmeetsingh @imjog @SIMRAN1 review the PR

**Comment by baquer** [comment]
As i run the app in simulator so mail, messages etc are missing in Screen shots
screen shot 2018-05-30 at 12 52 36 am
screen shot 2018-05-30 at 12 52 42 am

**Comment by baquer** [comment]
issue #56

**Comment by baquer** [comment]
@chashmeetsingh @imjog  please review the PR.

**Comment by jogendra** [comment]
@baquer Can you please post the screenshot of changes please?

**Comment by baquer** [comment]
bcd
abc

Here are the screenshots , i used UIActivityView for this task, thats why i closed the previous PR.

**Comment by baquer** [comment]
@imjog  here is the SS , all the issues of PR #56 is clear now.

File: FOSSAsia/MoreViewController.swift [hunk 1]
```diff
@@ -51,6 +51,20 @@ class MoreViewController: UITableViewController {
         case 3:
             self.present(self.createSVC(DefaultURLs.fossasiaGoogleGroups), animated: true, completion: nil)
             break
+        case 4:
+            let activityViewController : UIActivityViewController = UIActivityViewController(activityItems: [Constants.MoreViewController.message], applicationActivities: nil)
+            
+            activityViewController.setValue(Constants.MoreViewController.subject, forKey: Constants.MoreViewController.Title)
+            // For overcoming the crash in iPad
+            activityViewController.popoverPresentationController?.sourceView = self.view
+            activityViewController.excludedActivityTypes = [
```
Comment (jogendra) [inline FOSSAsia/MoreViewController.swift:60]: Why you excluded?

File: Podfile.lock [hunk 2]
```diff
@@ -49,4 +49,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 9f760f08d69dda4beb950e342f2a9b16dcf6e39c
 
-COCOAPODS: 1.5.2
+COCOAPODS: 1.5.3
```
Comment (jogendra) [inline Podfile.lock:52]: you should have updated cocoapods in different PR.

**Review by jogendra — CHANGES_REQUESTED** [review]

File: FOSSAsia/MoreViewController.swift (hunk 1 above)
Comment (baquer) [reply FOSSAsia/MoreViewController.swift:60]: Oops actually i messed up with that , Actually i want some of the things excluded from the activityVC such as airdrop , savetoPDF  etc .
i will update the changes

File: Podfile.lock (hunk 2 above)
Comment (baquer) [reply Podfile.lock:52]: Okay Will make Another PR for this.

**Review by jogendra — APPROVED** [review]
Looks good to me!

**Review by SIMRAN1 — APPROVED** [review]
Looks Nice to me!!

**Comment by baquer** [comment]
Due to  conflicts in More.storyboard, i am closing this PR , will open a new PR for this feature .

**Pull request closed without merging**